tests/test_other.py

```diff
@@ -7933,9 +7933,9 @@ def test(filename, expectations, size_slack):
 
```

Comment:
2 likes above this is `XXX wasm backend ignores EMSCRIPTEN_KEEPALIVE`, we can fix that line now with proper sizes (instead of None, and instead of the 0's at the end).